website/content/docs/what-is-vault.mdx

```diff
@@ -52,8 +52,8 @@ The key features of Vault are:
 
 - **Data Encryption**: Vault can encrypt and decrypt data without storing
   it. This allows security teams to define encryption parameters and
-  developers to store encrypted data in a location such as SQL without
-  having to design their own encryption methods.
+  developers to store encrypted data in a location such as an SQL database 
```

Comment:
```suggestion
  developers to store encrypted data in a location such as a SQL database 
```

Reply:
Thanks @taoism4504 . I did wonder why you originally changed 'a' to 'an' from my original commit. FYI, the discussion around `an` vs `a` before `SQL` and a list of other places in the docs where such inconsistencies exist can be found in [this comment](https://github.com/hashicorp/vault/pull/12274#issuecomment-942331973). Please consider fixing the others inconsistencies too.

Reply:
Yes, an oversight on my part, and will fix the inconsistencies. Thank you.